Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM golang:1.18-alpine AS builder
+RUN apk update && apk add --no-cache make git gcc libc-dev linux-headers python3-dev
+WORKDIR /nucleus
+COPY . /nucleus
+RUN rm -rf .git && git init
+RUN go mod tidy && go mod download
```

Comment:
> Updated some modules to accomodate importing into https://github.com/smk762/callisto/tree/cosmos/v0.46.x (required for explorer compatibility)

Can you add some more context about what exactly happens when we don't bump this dependencies?

Reply:
Thanks!